sdcm/nemesis.py

```diff
@@ -1523,6 +1523,79 @@ def disrupt_network_block(self):
         finally:
             self.target_node.traffic_control(None)
 
+    def disrupt_nodetool_removenode(self):
+        """
+        https://docs.scylladb.com/operating-scylla/procedures/cluster-management/remove_node/
+
+        1.terminate
+        2.Full repair
+        3.Nodetool removenode
+        4.Sleep 10 minutes
+        5.Add new node
+        """
+
+        self._set_current_disruption('TerminateAndRemoveNodeMonkey')
+        node_to_remove = self.target_node
+        up_normal_nodes = self.cluster.get_nodes_up_and_normal(verification_node=node_to_remove)
+        node = random.choice(up_normal_nodes)
+        # node_to_remove must be different than node
+        while node is node_to_remove:
+            node = random.choice(up_normal_nodes)
+
+        # node_to_remove must not be the only seed in cluster
+        num_of_seed_nodes = len(self.cluster.seed_nodes)
+        if node_to_remove.is_seed and num_of_seed_nodes < 2:
+            raise UnsupportedNemesis("To running seed decommission the cluster must contains at least 2 seed nodes")
+
+        # get node's host_id
+        removed_node_status = self.cluster.get_node_status_dictionary(
+            ip_address=node_to_remove.ip_address, verification_node=node)
```

Comment:
why do we need the node_status?

Reply:
out of `nodetool status` we need host_id...

Reply:
Yep, forgot we first terminate that node.